python/tk_multi_publish2/progress/progress_handler.py

```diff
@@ -50,15 +56,21 @@ def __init__(self, icon_label, status_label, progress_bar):
         self._icon_lookup = {
             self.PHASE_LOAD: QtGui.QPixmap(":/tk_multi_publish2/status_load.png"),
             self.PHASE_VALIDATE: QtGui.QPixmap(":/tk_multi_publish2/status_validate.png"),
+            self.PHASE_PRE_PUBLISH: QtGui.QPixmap(":/tk_multi_publish2/status_prepare.png"),
             self.PHASE_PUBLISH: QtGui.QPixmap(":/tk_multi_publish2/status_publish.png"),
-            self.PHASE_FINALIZE: QtGui.QPixmap(":/tk_multi_publish2/status_success.png"),
+            self.PHASE_FINALIZE: QtGui.QPixmap(":/tk_multi_publish2/status_finalize.png"),
```

Comment:
is there some icon we could use other than the green tick? it's confusing that the same one is used twice i think.

Finalize is typically used to version up, to prepare the system for further work once the publish has been carried out. Wonder if it could be an icon relating to that? some page being turned, some thing to indicate 'next step', or something like a thumbs up? Not sure myself.... :|

Reply:
Something like the step into icon on the pycharm debugger?